src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { memo, useEffect } from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  error: string;
+  onClosingErrorMessage: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const ErrorNotification: React.FC<Props> = memo((props) => {
+  const { error, onClosingErrorMessage } = props;
+
+  useEffect(() => {
```

Comment:
taking into account this comment: https://github.com/mate-academy/react_todo-app-loading-todos/pull/280#discussion_r1096535331, you should return the function inside this callback, where you clear timeout:
![image](https://user-images.githubusercontent.com/86923089/216814686-e2b1bc81-e926-4977-b416-1807915eced6.png)

Reply:
Thank you! Done✅

Reply:
comments to clearTimeOut https://mate-parttime.slack.com/archives/C048S1Q5C94/p1675596276867079